### DISCUSSION TITLE
App router in nextjs doesn't render page.js and layout.js at the same time (only one of those)

### DISCUSSION THREAD

**Discussion by Username1111111111** [vercel/next.js#62897] [Help] [answered]
### Summary

I have a project and I've noticed that at some routes there page.js with layout js doesn't work together.

Here are project structure and discussed folder: [Link](https://github.com/Username1111111111/SeTTTify/tree/main/app/collection)

It's app/collection/[collectionId]/edit and along the way there are dummy page.js and layout.js at different places. 

For some strange reason they don't work together. If I would remove layout.js file - the page.js file would render. The console and everything doesn't show errors. 

Is it a bug or I completely don't understand some principle of app router? (For example, layout is for sharing components to child components in route, page.js is only for a segment of route in which it placed)

Or, maybe, there could be only one layout.js at the topmost route /  and it conflicting with those down the route?

### Example

https://github.com/Username1111111111/SeTTTify/tree/main/app/collection

**Comment by ruslanyar** [comment, marked as answer]
Hi! It's not a bug.
Each layout should accept children.

[docs](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts#nesting-layouts)

**Reply by Username1111111111** [reply]
Thank you! Will check how to wrap children and if it will work as intended
